tools/mapping/gen_domain_files/src/Makefile

```diff
@@ -70,15 +70,15 @@ ifeq ($(SMP),$(null))
   SMP := FALSE
 endif
 
-CPPDEF += $(USER_CPPDEFS)
+CPPDEFS += $(USER_CPPDEFS) -D$(OS)
 
 # Set optimization on by default
 ifeq ($(OPT),$(null))
   OPT := TRUE
 endif
 
 ifeq ($(OPT),TRUE)
-  CPPDEF := -DOPT
+  CPPDEFS := -DOPT
```

Comment:
This needs to be += so it doesn't throw everything away...
So...

CPPDEFS += -DOPT